core/src/main/scala/kafka/utils/Json.scala

```diff
@@ -78,4 +79,11 @@ object Json {
    */
   def encodeAsBytes(obj: Any): Array[Byte] = encode(obj).getBytes(StandardCharsets.UTF_8)
 
+  def encodeToJsonString(obj: Map[String,Any]): String = {
```

Comment:
This should be `encodeAsString` and it should take `Any` as the type. We can maybe rename the existing `encode` to `legacyEncodeAsString`.

Reply:
This change is done.